mime-ballerina/build.gradle

```
@@ -227,8 +251,8 @@ task createArtifactZip(type: Zip) {
 
 publishing {
     publications {
-        mavenJava(MavenPublication) {
-            artifact source: createArtifactZip, extension: 'zip'
+        maven(MavenPublication) {
+            artifact source: artifactCodeCoverageReport, classifier: 'jacoco'
```

We need both the artifacts

```suggestion
            artifact source: createArtifactZip, extension: 'zip'
            artifact source: artifactCodeCoverageReport, classifier: 'jacoco'
```